tests/components/vesync/snapshots/test_diagnostics.ambr

```diff
@@ -0,0 +1,274 @@
+            'auto_target_humidity': 60,
+            'automatic_stop': True,
+            'display': True,
+          }),
+          'config_dict': dict({
+            'features': list([
+              'warm_mist',
+              'nightlight',
+            ]),
+            'mist_levels': list([
+              1,
+              2,
+              3,
+              4,
+              5,
+              6,
+              7,
+              8,
+              9,
+            ]),
+            'mist_modes': list([
+              'humidity',
+              'sleep',
+              'manual',
+            ]),
+            'models': list([
+              'LUH-A602S-WUSR',
+              'LUH-A602S-WUS',
+              'LUH-A602S-WEUR',
+              'LUH-A602S-WEU',
+              'LUH-A602S-WJP',
+            ]),
+            'module': 'VeSyncHumid200300S',
+            'warm_mist_levels': list([
+              0,
+              1,
+              2,
+              3,
+            ]),
+          }),
+          'config_module': 'WFON_AHM_LUH-A602S-WUS_US',
+          'connection_status': 'online',
+          'connection_type': 'WiFi+BTOnboarding+BTNotify',
+          'current_firm_version': None,
+          'details': dict({
+            'automatic_stop_reach_target': True,
+            'display': False,
+            'humidity': 35,
+            'humidity_high': False,
+            'mist_level': 3,
+            'mist_virtual_level': 9,
+            'mode': 'humidity',
+            'night_light_brightness': 0,
+            'warm_mist_enabled': False,
+            'warm_mist_level': 0,
+            'water_lacks': False,
+            'water_tank_lifted': False,
+          }),
+          'device_image': 'https://image.vesync.com/defaultImages/LV_600S_Series/icon_lv600s_humidifier_160.png',
+          'device_name': 'Humidifier',
+          'device_status': 'off',
+          'device_type': 'LUH-A602S-WUS',
+          'enabled': False,
+          'extension': None,
+          'features': list([
+            'warm_mist',
+            'nightlight',
+          ]),
+          'mac_id': '00:00:00:00:00:00',
+          'manager': '**REDACTED**',
+          'mist_levels': list([
+            1,
+            2,
+            3,
+            4,
+            5,
+            6,
+            7,
+            8,
+            9,
+          ]),
+          'mist_modes': list([
+            'humidity',
+            'sleep',
+            'manual',
+          ]),
+          'mode': None,
+          'night_light': True,
+          'speed': None,
+          'sub_device_no': None,
+          'type': 'wifi-air',
+          'uuid': '00000000-1111-2222-3333-444444444444',
+          'warm_mist_feature': True,
+          'warm_mist_levels': list([
+            0,
+            1,
+            2,
+            3,
+          ]),
+        }),
+      ]),
+      'outlets': list([
+      ]),
+      'switches': list([
+      ]),
+    }),
+    'vesync': dict({
+      'account_id': '9999999',
```

Comment:
should this be redacted ?

Reply:
@balloob, are there any guidelines for what should be redacted? You've highlighted uuid, mac_id, and account_id. I don't know how uuid and mac_id are used, so I have no problem redacting them. However, I made a conscious decision to leave account_id visible to allow the user to validate that they were using the correct account. This could have been faulty logic on my part.

Reply:
I think that anything that is user identifiable, like IDs, should be redacted as much as possible. I doubt a user will check this value. If it's not use to you for debugging, let's redact it .

Reply:
Understood, account_id removed from output